UI Improvements

MeijerM1 opened the pull request:
This PR implements the new designs based on the [InVision designs](https://projects.invisionapp.com/d/main?origin=v7#/projects/prototypes/20080254?scrollOffset=218).

Since running the application locally can be a bit of hassle (you need to run the backend to actually do anything). The plan is to use this PR to review the code. When it is on staging we will do an actual review of the UI itself.

RoelN commented on src/components/Circle/Circle.tsx:
```diff
@@ -15,16 +15,20 @@ function CustomCircle(props: Props): React.ReactElement {
-          <span data-testid="goal-kudos">of {props.neededKudos}₭ for {props.goal}</span>
+          <h2 data-testid="current-kudos" className={s.current}>
+            {props.currentKudos}
+            <span style={{ fontSize: '35px', margin: 0, padding: 0 }}>₭</span>
```
Why inline styles here?

RoelN commented on src/components/navigation/Desktop.module.scss:
```diff
@@ -2,8 +2,12 @@
   position: fixed;
   top: 0;
   width: 100%;
+  background: #00749A;
```
Nit: lowercase

RoelN commented on src/components/navigation/Desktop.module.scss:
```diff
@@ -2,8 +2,12 @@
+:global(#root) {
+  .menu {
+    height: 62.84px;
+    background: #00749A;
```
Nit: lowercase (also, ugh that magic number!

RoelN commented on src/components/navigation/Desktop.tsx:
```diff
@@ -34,9 +34,14 @@ export interface Props {
         <Container>
-          <Menu.Item data-testid="home-button" onClick={() => props.history.push(PATH_FEED)}>Home</Menu.Item>
+          <Menu.Item
+            style={{ borderLeft: 'none' }}
```
Why inline?

RoelN commented on src/components/navigation/Toolbar.module.scss:
```diff
@@ -15,10 +20,15 @@
+  position: static;
   height: 68px;
-  background-color: white;
+  background-color: #00749A;
```
Nit: lowercase

RoelN commented on src/components/navigation/Toolbar.module.scss:
```diff
@@ -15,10 +20,15 @@
   flex-direction: row;
+
+  span {
+    color: white;
```
Any reason to use a mix of `white` and `#ffffff`?

RoelN commented on src/modules/choose-team/CreateTeamPage.tsx:
```diff
@@ -92,56 +93,70 @@ class CreateTeamPage extends Component<Props, State> {
-        </div>
+        <Responsive minWidth={Responsive.onlyTablet.minWidth}>
+          <Segment
+            style={{
```
Why inline?

RoelN commented on src/modules/choose-team/components/ChooseTeam.module.scss:
```diff
@@ -1,3 +1,3 @@
 .button {
-  margin: 0 !important;
+  margin: 1rem;
```
Nice, good riddance of that `!important` 👍

RoelN commented on src/modules/feed/FeedPage.module.scss:
```diff
@@ -1,22 +1,73 @@
+    font-family: Rajdhani, sans-serif;
+    font-weight: lighter;
+    font-size: 34px;
+    letter-spacing: 0;
```
This explicitly disables justification of text. Maybe you want `normal` here?

RoelN commented on src/modules/manage-team/ManageTeamPage.module.scss:
```diff
@@ -0,0 +1,21 @@
+    a {
+      width: 100%;
+      border: none;
+      border-radius: 0 !important;
```
No way around using `!important`?

RoelN commented on src/modules/manage-team/ManageTeamPage.module.scss:
```diff
@@ -0,0 +1,21 @@
+  }
+
+  .column {
+    padding-top: 0 !important;
```
No way around using `!important`?

RoelN commented on src/modules/statistics/Statistics.module.scss:
```diff
@@ -13,3 +26,19 @@
+}
+
+.current_percentage_text {
+  float: right;
```
Maybe use `flex` to align, as with other elements in the app?

RoelN commented on src/modules/statistics/Statistics.tsx:
```diff
@@ -41,19 +43,12 @@ export interface ActiveGoal {
-    <h4 style={{ marginTop: '0.6rem' }}>{moment().format('MMMM Do, YYYY')}</h4>
+  <div className={s.container}>
+    <h2 className={s.kudo_header}>₭udometer</h2>
+    <h4 style={{ color: '#FFF', marginTop: '0.6rem' }}>{moment().format('MMMM Do, YYYY')}</h4>
```
Looks like the hierarchy is off here? h2, h4, then h3?

MeijerM1 commented:
@RoelN I removed all the inline styles, extracted all color usage to css vars (I build a very, very simple theme file) and I removed all the importants where possible, in some places Semantic UI actually uses it so I need to use it as well to override it. 

I'd like to get rid of the magic numbers used in all the media queries but because of React CSS Modules I can't have a single file that defines all these variables and import that before all the other styles. 

One approach would be to define a single file with only variables in it and import that file in every `.scss` file, but that seems like a code smell to me.

RoelN commented on src/modules/feed/FeedPage.module.scss:
```diff
@@ -1,22 +1,95 @@
+
+  .board_container {
+    max-width: 480px;
+    -webkit-overflow-scrolling: touch;
```
Small nit: if you use prefixed properties, leave a quick comment why this is only needed for `-webkit`

RoelN commented:
@MeijerM1 Re: variables, maybe you can use a global stylesheet that isn't scoped to a component, and declare them there?

Otherwise, nice job, and thanks for cleaning up!
